MODULE.bazel

```diff
@@ -33,9 +33,78 @@ bazel_dep(
 bazel_dep(name = "bazel_skylib", version = "1.8.1", dev_dependency = True)
 bazel_dep(name = "fmt", version = "11.0.2", dev_dependency = True)
 bazel_dep(name = "rules_python", version = "1.6.0", dev_dependency = True)
+bazel_dep(name = "toolchains_llvm", version = "1.5.0", dev_dependency = True)
 
 # Configuration of above deps.
 
+# Configure and register the LLVM toolchain.
+
+# Since the clang toolchain we're using doesn't let us extract the default flags, we have to
+# manually specify the default flags here. These are copied from:
+# https://github.com/bazel-contrib/toolchains_llvm/blob/069ee4e20ec605a6c76c1798658e17175b2eb35e/toolchain/cc_toolchain_config.bzl#L118
+BASE_CLANG_COPTS = [
+    # Security
+    "-U_FORTIFY_SOURCE",  # https://github.com/google/sanitizers/issues/247
+    "-fstack-protector",
+    "-fno-omit-frame-pointer",
+    # Diagnostics
+    "-fcolor-diagnostics",
+    "-Wall",
+    "-Wshadow",
+    "-Wthread-safety",
+    "-Wself-assign",
+]
+
+# Extra flags we want to pass to the compilers.
+EXTRA_COPTS = [
+    "-Wextra",
+    "-pedantic",
+]
```

Comment:
This is now duplicated with `copts.bzl`.  Can we de-duplicate?

Alternatively, if `copts.bzl` is about to be deleted in one of your upcoming PRs anyway, then I'm fine with leaving the duplication in place.

Reply:
This will be deduplicated when the GCC toolchain comes in.  You can't do a `load` in the `MODULE.bazel`, so no easy way to share this with `WORKSPACE`.